processor/transformer/transformer.go

```diff
@@ -29,6 +29,7 @@ const (
 	EventFilterStage            = "event_filter"
 	DestTransformerStage        = "dest_transformer"
 	TrackingPlanValidationStage = "trackingPlan_validation"
+	DROP_STATUS_CODE            = 722
```

Comment:
```suggestion
	DropStatusCode            = 722
```

> See https://go.dev/doc/effective_go#mixed-caps. This applies even when it breaks conventions in other languages. For example an unexported constant is `maxLength` not `MaxLength` or `MAX_LENGTH`.

https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps